org.qt.photoqt.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "org.qt.photoqt",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "command": "photoqt",
+    "rename-desktop-file": "photoqt.desktop",
+    "rename-appdata-file": "photoqt.appdata.xml",
+    "rename-icon": "photoqt",
+    "finish-args": ["--share=ipc","--socket=x11","--socket=wayland","--filesystem=host","--device=dri","--socket=pulseaudio"],
```

Comment:
Please put these on their own line. Same applies for everything else and broken whitespace throughout.